src/hypothesis/settings.py

```diff
@@ -278,7 +278,7 @@ def __exit__(self, *args, **kwargs):
 
 Settings.define_setting(
     u'max_examples',
-    default=200,
+    default=int(os.getenv(u'HYPOTHESIS_MAX_EXAMPLES', 200)),
```

Comment:
The more idiomatic way to do this is `os.environ.get(u'HYPOTHESIS_MAX_EXAMPLES', 200))` IMHO.